docs/install/Knative-with-OpenShift.md

```diff
@@ -5,219 +5,47 @@ weight: 15
 type: "docs"
 ---
 
-This guide walks you through the installation of the latest version of
-[Knative Serving](https://github.com/knative/serving) on an
-[OpenShift](https://github.com/openshift/origin) using pre-built images and
-demonstrates creating and deploying an image of a sample "hello world" app onto
-the newly created Knative cluster.
+# Knative Install on OpenShift
+
+This guide walks you through the installation of the latest version of [Knative
+Serving](https://github.com/knative/serving) on [OpenShift](https://github.com/openshift/origin) by using the Knative Serving Operator. The operator is available on the OpenShift OperatorHub. After you install Knative Serving, create and deploy an image of a sample "hello world" app onto the newly created Knative cluster.
 
 You can find [guides for other platforms here](./README.md).
 
 ## Before you begin
 
-These instructions will run an OpenShift 3.11 (Kubernetes 1.11) cluster on your
-local machine using
-[`oc cluster up`](https://docs.openshift.org/latest/getting_started/administrators.html#running-in-a-docker-container)
-to test-drive knative.
-
-## Install `oc` (openshift cli)
-
-You can install the latest version of `oc`, the OpenShift CLI, into your local
-directory by downloading the right release tarball for your OS from the
-[releases page](https://github.com/openshift/origin/releases/tag/v3.11.0).
+* An OpenShift 4 cluster is required for installation. Visit [try.openshift.com](try.openshift.com) for information on setting up a cluster. You will need cluster administrator privileges to install and use Knative on an OpenShift cluster. 
 
-```shell
-wget https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz -o oc.tar.gz
-tar xzvf openshift*tar.gz
-mv openshift-origin-client-tools-*/oc .
-mv openshift-origin-client-tools-*/kubectl .
-rm -rf openshift-origin-client-tools-*/
-```
+## Installing the Knative Serving Operator
 
-You will now have the `oc` (and `kubectl`) binaries in your local directory.
+1. Go to **Catalog > OperatorHub** in the OpenShift Web Console. A list of operators for OpenShift, provided by Red Hat as well as a community of partners and open-source projects is provided. Click on the **Knative Serving Operator** tile.
 
-## Scripted cluster setup and installation
+> **NOTE:** Use the **Filter by Keyword** box to facilitate the search of the Knative Serving operator in the catalog.
 
-For Linux and Mac, you can optionally run a
-[script](./scripts/knative-with-openshift.sh) that automates the steps on this
-page.
+![KSO Tile](images/knative_serving_tile_highlighted.png)
 
-Once you have `oc` present on your machine and in your `PATH`, you can simply
-run [this script](./scripts/knative-with-openshift.sh); it will:
+2. A **Show Community Operator** dialog box will open. Click **Continue** to proceed.
```

Comment:
I know the automatic numbering is not always working out correctly on the website, but we do want to stick to using 1. for each numbered item in a list. At some point the automatic numbering will be fixed, and we won't have to go back and update a lot of static numbers.